src/Tools/ExternalAccess/CompilerDeveloperSDK/Microsoft.CodeAnalysis.ExternalAccess.CompilerDeveloperSDK.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net7.0</TargetFrameworks>
```

Comment:
curious: why `net7.0` here? That is a short release. Would assume the server was on `net6.0`. Is that assumption wrong?

Reply:
That assumption is wrong: https://github.com/dotnet/roslyn/blob/b059755627eb8ee45aee083c967bada95583bbfc/src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj#L4